src/components/pages/auth/logout.tsx

```diff
@@ -31,18 +31,25 @@ export function Logout({ oauth }: { oauth?: boolean }) {
   useEffect(() => {
     checkInstance({ redirect: false })
 
-    const redirection = filterUnwantedRedirection({
-      desiredPath: sessionStorage.getItem('previousPathname'),
-      unwantedPaths: [settingsUrl, loginUrl, registrationUrl],
-    })
+    const cookieSession = AuthSessionCookie.parse()
```

Comment:
Is `AuthSessionCookie.parse()` the same as `AuthSessionCookie.get()`?

Reply:
- `get` returns a `string` 
- `parse` returns a `Session` object (and you can optionally supply a cookie yourself)
But the difference is basically just a `JSON.parse`